test/e2e/specs/editor/various/navigable-toolbar.spec.js

```diff
@@ -1,23 +1,26 @@
 /**
  * WordPress dependencies
  */
-import { createNewPost, pressKeyWithModifier } from '@wordpress/e2e-test-utils';
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
-async function isInBlockToolbar() {
-	return await page.evaluate( () => {
-		return !! document.activeElement.closest(
-			'.block-editor-block-toolbar'
-		);
-	} );
-}
+test.use( {
+	navigableToolbar: async ( { page }, use ) => {
+		await use( new navigableToolbar( { page } ) );
+	},
+} );
 
-describe( 'Block Toolbar', () => {
-	beforeEach( async () => {
-		await createNewPost();
+test.describe( 'Block Toolbar', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
 	} );
 
-	describe( 'Contextual Toolbar', () => {
-		it( 'should not scroll page', async () => {
+	test.describe( 'Contextual Toolbar', () => {
+		test( 'should not scroll page', async ( {
+			page,
+			pageUtils,
+			navigableToolbar,
+		} ) => {
+			const wp = '';
```

Comment:
Was this related to a `wp` is undefined kind of linting error? If so, I think it's that the places that use `wp` need to be `window.wp` and you can remove this `const wp` line.